tutorials/elm-ui/README.md

```diff
@@ -0,0 +1,718 @@
+_you_ - the conscientious developer/designer - learn CSS.
+Unless you can ensure that _everyone_ in your team/company
+has the same level of skill and care in their work,
+it's _inevitable_ that over-time
+duplicate and _conflicting_ styles will be added.
+
+<div align="center">
+  <a href="https://twitter.com/100soft">
+      <img src="https://media.giphy.com/media/Jrl4FlTaymFFbNiwU5/giphy.gif"
+        width="500" alt="dumpster fire">
+  </a>
+</div>
+
+Take any popular website and run it through CSS Stats https://cssstats.com
+to see how many styles are being used. <br />
+A good example is
+[Twitter.com](https://cssstats.com/stats?url=https%3A%2F%2Ftwitter.com).
+Their CSS is **632KB** (_or **113KB** when compressed+gzipped_). <br />
+https://cssstats.com/stats?url=https%3A%2F%2Ftwitter.com
+
+![twitter-css-stats](https://user-images.githubusercontent.com/194400/70567485-f56b2580-1b8d-11ea-94c4-4e2ca8b5cd10.png)
+
+That might not sound like a lot,
+but for context **632KB** is enough to write a 100k+ word novel! <br />
+The point is: the developers/designers at Twitter - a tech company
+with **billions** in revenue and some of the most talented people -
+have _lost control_ over their CSS.
+
+> **Twitter** created **Bootstrap**
+"_the most popular front-end framework in the world_".
+**Bootstrap	`v4`** is **123kb** (_uncompressed_)
+and **23kb** (_compressed+gzipped_);
+the _whole_ library including grid layout,
+typography, colors, all components, forms and utilities.
+Bootstrap is meant to be a _complete_ responsive grid-based design system
+that _anyone_ can use to build their App/website.
+And yet twitter, which has a _great_ in-house design team
+with their own _system_ and component library,
+have managed to create a **CSS monster**
+that is **5 times** the size of all of **Bootstrap**.
+Even if they had **20kb** of colors,
+that's enough for 256 colors, it would still be less than **150kb**.
+
+
+The problem isn't you,
+or "other people",
+the problem is the (_lack of a_) _system_.
+If the _system_ you are using to create your layout/styles
+does not give you any consistency guarantees at compilation time,
+then you don't _have_ a good enough system and chaos is _guaranteed_.
+
+<br />
+
+### Drawbacks?
+
+So far in using `elm-ui` we have only spotted one potential drawback.
+All `Colors` (_for text, borders or backgrounds_)
+have to be specified in terms of **rgb** values.
+i.e. you need to supply the individual values for `Red` `Blue` `Green` as `Int`.
+e.g:
+```elm
+Background.color (rgb255 75 192 169)
+```
+
+If we attempt to invoke `rgb255 75, 192, 169`
+we will see the following error:
+
+![elm-ui-rgb-error](https://user-images.githubusercontent.com/194400/70663222-cfae5100-1c5f-11ea-8fa6-6d4437e3af14.png)
+
+We feel this is a _comparatively small_ price to pay
+given that it means colors can be checked at compile time.
+It allows us to _easily_
+implement a color picker allowing people to easily select _any_ color
```

Comment:
```suggestion
implement a color picker allowing people to smoothly select _any_ color
```

Reply:
Took all my might not to amend `color` to the British spelling 😆

Reply:
Britain _lost_ https://en.wikipedia.org/wiki/American_Revolutionary_War so it's ***`color`***.